### DISCUSSION TITLE
Add Whisper API to ai-chatgpt Example?

### DISCUSSION THREAD

**Discussion by JacobGoldenArt** [vercel/examples#588] [Ideas]
I'm super impressed with the ai-chatgbt example. I haven't worked with streaming before and I'm still wrapping my head around it but wow, nice work. I'd love to see a fork that uses the Whisper API as input. I've been looking into using it. I'd love to see an example that uses WebAudio and maybe some more of that streaming magic to fetch the transcription and feed that as an input into chatgbt. : ) Fun times!

**Comment by dominiksipowicz** [comment]
Thank you. Good idea!
